Unity-Technologies/ScriptableRenderPipeline
# Hdrp/add documentation link

### Purpose of this PR
Add documentation links in component's Inspector header bar (HDRP + some Core preparation)
Also added unit test on links (see below).

*Known issue:*
- This require the documentation to make every documentation page live in the same namespace than the actual component (so moving the Volumes in Core with inside the webpage links for HDRP and URP etc.)
- The links of built-in component Camera, Light and ReflectionProbe that we only overide the inspector cannot be updated. The acurate link will be on the corresponding AdditionalData component for now.

*Workflow change for release:*
- When releasing, if documentation version need to be updated (it need to be done for all page at target version), change the Documention.version. Documentation is a file at base of Runtime folder of each package (HDRP and CoreRP for now)

---
### Release Notes
Fixed documentation links in component's Inspector header bar.

---
### Testing status
**Katana Tests**: https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?ScriptableRenderLoop_branch=HDRP%2Fadd-documentation-link&automation-tools_branch=master&unity_branch=trunk&sort=1-asc

**Manual Tests**: What did you do?
- [x] Opened test project + Run unit tests

**Automated Tests**: 
- Added Unit test grabbing all link and testing if they are accessible for CoreRP (but disabled it for now as documentation link are not done yet in CoreRP)
- Added Unit test grabbing all link and testing if they are accessible for HDRP

---
### Overall Product Risks
**Technical Risk**: Low
**Halo Effect**: Low

---
### Comments to reviewers
@iTris666 for notification

*QA:*
- The documentation link should go to 6.9 documentation link (the latest released documentation at the moment) See also *known issue* above.
